src/plugins/ai_assistant_management/observability/public/routes/components/languages.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const languages = [
```

Comment:
This list is extract from https://github.com/L-P/native-language-list/blob/master/data/langs.json, not sure if that's allowed, how to credit it or if there is a better way to get this list.

Reply:
I'd expect the language to be specified by the [ISO 639 code](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) and then map it to a human readable label (like this one)

Reply:
Btw. the link json file is indeed a mapping between iso code and label. Should we do the same?

Reply:
... one more note: this is something we at least eventually should be getting from the platform and not provide ourselves. I just did a quick search and didn't find anything exhaustive so fine to add this here for now

Reply:
Can you expand on why you would expect us to store the mapping, rather than just the list of languages? 

I pinged you in Slack regarding the "getting from platform" part, it's not something they offer today and it seems it's not something they would likely offer since it's a fairly unique use case (at least right now).

Reply:
I'm probably just used to dealing with iso codes when specifying languages. But maybe it's better to use the human readable label than an iso code when working with the LLM

Reply:
Let's leave it as-is.